### DISCUSSION TITLE
Has development on this project been discontinued?

### DISCUSSION THREAD

**Discussion by marianoibarra** [master-co/css#371] [Question]
Hello,

I’ve been following this project and find it incredibly useful. However, I noticed that the last commit was made approximately four months ago, and there haven’t been any recent updates or communication from the maintainers.

I was wondering if development on this project has been paused or discontinued. If so, would it be possible to know if there are plans for future updates or if the community could take over maintenance?

Thank you for your time, and I appreciate all the work that has gone into this project so far!

Looking forward to hearing from you.

**Comment by 1aron** [comment]
@marianoibarra Thank you for your concern and support. 😄 At this moment, we are reigniting our efforts. Over the past few months, we’ve been deeply engaged in private projects, which has delayed progress on Master.

We hope that in the future, we can continue to grow and eventually shift our full focus back to Master.
